examples/main/main.cpp

```diff
@@ -485,6 +509,7 @@ bool output_json(struct whisper_context * ctx, const char * fname, const whisper
                         value_i("from", t0 * 10);
                         value_i("to", t1 * 10, true);
                     end_obj();
+                    value_s("speaker", speaker);
```

Comment:
Shouldn't this only be set if `params.diarize` is true?

Reply:
yes but we always diarize

This fork is for our purposes, not for generic whisper.cpp usage